src/Extend/Theme.php

```diff
@@ -52,6 +53,54 @@ public function overrideFileSource(string $file, string $newFilePath, string $ex
         return $this;
     }
 
+    /**
+     * This method allows you to add custom Less functions.
+     *
+     * **Example usage:**
+     * ```php
+     * (new Extend\Theme)
+     *     ->addCustomLessFunction('is-flarum', function (mixed $text) {
+     *         return strtolower($text) === 'flarum'
```

Comment:
I feel like we should document which return types are valid, and maybe specify that in the docblock?

Reply:
Good idea